kpi/views/tmp_browsers_kpi_2023.view.lkml

```diff
@@ -0,0 +1,116 @@
+view: tmp_browser_kpis_2023 {
+  derived_table: {
+    sql:
+      WITH
+      desktop_dau AS (
+      SELECT submission_date,
+        'DAU' AS metric,
+        'desktop' AS product,
+        COUNTIF(
+          active_hours_sum > 0 AND
+          COALESCE(
+            scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+            scalar_parent_browser_engagement_total_uri_count_sum,
+            0
+          ) > 0
+        ) AS value
+      FROM `mozdata.telemetry.clients_daily`
```

Comment:
Do we want to use the the "shredded" numbers until July?
Before 2023-03-05 when shredder last ran, the numbers are not a 1:1 match with the active_users_aggregates tables. After that date, the [numbers between aggregates and clients_daily actually match exactly](https://docs.google.com/document/d/1J6VPYXin7Cf9aEItMadFLGooTmCaHhhTUSCaGTNcBlY/edit#heading=h.y2jiq1hzphef). The concern is that it generates confusion when we move to using the aggregates and DAU is different.

Reply:
We chatted offline and decided to communicated this difference when we roll over to `active_users_aggregates`.

The `clients_daily` table gets updated with "shredded" numbers automatically, but the `active_users_aggregates` tables do not account for shredded `client_ids` unless they're backfilled. This can actually be preferable, because it will mean that historic numbers won't change over time.